HW_1/Task_1.py

```diff
@@ -0,0 +1,327 @@
+'''1. За день машина проезжает n километров.  
+Сколько дней нужно, чтобы проехать маршрут длиной m километров?  
+При решении этой задачи нельзя пользоваться условной инструкцией if и циклами.
+
+**Input:**
+
+n = 700
+
+m = 750
+
+**Output:**
+
+2'''
+
+
+def distance():
+    print('Задание 1')
+    n = int(input())
+    m = int(input())
+    print((m-1)//n+1)
+
+# distance()
```

Comment:
Совсем ни к чему тут использовать функцию. Не принято определять функцию, если она вызывается единожды.
Если уж вы определяете функцию, то делайте ее чистой. То есть без побочных эффектов и детерминированной. К тому же не забывать о приводить к PEP8.
```suggestion
def distance(n, m):
    return (m - 1) // n + 1

n = int(input())
m = int(input())
print(f'Задание 1: {distance(n, m)}')
```